bootstrap/frontend/src/main/scala/io/udash/bootstrap/datepicker/UdashDatePicker.scala

```diff
@@ -164,32 +165,44 @@ class UdashDatePicker private[datepicker](val date: Property[ju.Date],
     )
   }
 
-  private def dateToMomentString(date: ju.Date): String =
-    Try(moment(options.get.locale.getOrElse("en"), date.getTime, "x").format(options.get.format)).getOrElse(null)
+  private def internalFormat = options.get.format
+  private def internalLocale = options.get.locale.getOrElse("en")
+
+  private def dateToMomentString(date: Option[ju.Date]): String =
+    date.map(d => moment(internalLocale, d.getTime, "x").format(internalFormat)).getOrElse("")
 
   private def dateToMoment(date: ju.Date): MomentFormatWrapper =
-    Try(moment(options.get.locale.getOrElse("en"), date.getTime, "x")).getOrElse(null)
+    Try(moment("en", date.getTime, "x")).getOrElse(null)
```

Comment:
Out of 3 moment converters this one uses fixed EN locale. Is this intentional? I'd rather they all shared the same external lib call.